vision/standard_object/odlc_text_detection.py

```diff
@@ -0,0 +1,155 @@
+"""
+Method for reading the text on the odlc standard objects.
+"""
+
+from typing import Any
+
+import cv2
+import numpy as np
+import numpy.typing as npt
+import pytesseract
+
+from vision.common.bounding_box import ObjectType, BoundingBox
+from vision.common.constants import Image
+
+
+def text_detection_pre_processing(unprocessed_img: Image) -> Image:
+    """
+    This does the preprocessing effects on the passed image needed to make text detection easier.
+
+    Parameters
+    ----------
+    unprocessed_img: Image
+        The image that you want to do the preprocessing on.
+
+    Returns
+    -------
+    final_img: Image
+        The image after all of the preprocessing effects have been run.
+    """
+
+    # Contrast and brightness
+    contrast_img: Image = cv2.convertScaleAbs(unprocessed_img, alpha=1.25, beta=-30)
+
+    # Grayscale
+    grayscale_img: Image = cv2.cvtColor(contrast_img, cv2.COLOR_RGB2GRAY)
+
+    # Blur
+    blured_img: Image = cv2.medianBlur(grayscale_img, ksize=3)
+
+    # Erode and dilate
+    kernel: Image = np.ones((5, 5), np.uint8)
+    eroded_img: Image = cv2.erode(blured_img, kernel=kernel, iterations=1)
+    final_img: Image = cv2.dilate(eroded_img, kernel=kernel, iterations=1)
+
+    return final_img
+
+
+def crop_image(uncropped_img: Image, bounds: BoundingBox) -> Image:
```

Comment:
Remove this and use the new `crop_image()` function in `vision/common/crop.py`. (You will need to perform a `git pull origin develop` or something similar to update your local files with the new changes)